integration-tests/main/src/test/java/io/quarkus/it/main/OpenApiTestCase.java

```diff
@@ -90,7 +90,8 @@ public void testOpenAPIJSON() throws Exception {
         Assertions.assertEquals(1, keys.size());
         Assertions.assertEquals("get", keys.iterator().next());
 
-        String uniSchemaType = schemaType("200", DEFAULT_MEDIA_TYPE, uniObj.getJsonObject("get").getJsonObject("responses"),
+        String uniSchemaType = schemaType("200", DEFAULT_MEDIA_TYPE_PRIMITAVE,
```

Comment:
Not worth fixing here as it was preexisting but next time you update this test:
```suggestion
        String uniSchemaType = schemaType("200", DEFAULT_MEDIA_TYPE_PRIMITIVE,
```